bigchaindb/voter.py

```diff
@@ -190,3 +190,69 @@ def start(self):
         p_validate.start()
         p_vote.start()
         p_update.start()
+
+
+class Election(object):
+
+    def __init__(self, q_block_new_vote):
+        """
+        Initialize the class with the needed queues.
+
+        Initialize a queue where blocks with new votes will be held
+        """
+        self.q_block_new_vote = q_block_new_vote
+        self.q_blocks_with_quorum = mp.Queue()
+        self.q_invalid_blocks = mp.Queue()
+
+    def check_for_quorum(self):
```

Comment:
I would move most of the logic in this method to `bigchaindb.core.Bigchain`.
It would be handy to have a method in which you give it a block (any block) and it returns either `valid`, `invalid`, or undecided.
We will need to use this functionality a lot.